etc/dependency-check-suppression.xml

```diff
@@ -123,4 +123,29 @@
    <vulnerabilityName>CVE-2020-15250</vulnerabilityName>
 </suppress>
 
+<!-- grpc -->
```

Comment:
This is unrelated to this PR

Reply:
Yes, this is my fault. I copied master version here to remove all my changes. Unfortunately someone changed it since I made my fork.